delegation-toolkit/get-started/quickstart.md

```diff
@@ -17,157 +16,77 @@ by creating a delegator account and completing the delegation lifecycle (creatin
 
 ### 1. Set up a Public Client
 
-Set up a [Viem Public Client](https://viem.sh/docs/clients/public) using Viem's `createPublicClient` function.
-This client will let the delegator account query the signer's account state and interact with smart contracts.
+Set up a [Viem Public Client](https://viem.sh/docs/clients/public) using Viem's `createPublicClient` function. This client will let the smart account query the signer's account state and interact with blockchain network.
 
 ```typescript
-import { createPublicClient, http } from "viem";
-import { sepolia as chain } from "viem/chains";
- 
+import { createPublicClient, http } from 'viem'
```

Comment:
```suggestion
import { createPublicClient, http } from "viem"
```

We try to use double quotes consistently throughout the doc site. This applies to the rest of the code samples on this page.

Reply:
I think it's my habit of saving files frequently, and since Prettier is configured to use single quotes, it keeps formatting them that way. Maybe we should consider changing the config?

Prettier configuration: https://github.com/MetaMask/metamask-docs/blob/main/.prettierrc